### PR TITLE
Add large tests for trigger APIs

### PR DESCRIPTION
**Pull request by yongpingchen**
Add large tests to test the following APIs:
- ThingIFAPI#postNewTrigger(TriggeredCommandForm, Predicate, TriggerOptions)
- ThingIFAPI#postNewTrigger(TriggeredCommandForm, Predicate)
- ThingIFAPI#postNewTrigger(ServerCode, Predicate, TriggerOptions)
- ThingIFAPI#postNewTrigger(ServerCode, Predicate, TriggerOptions)
- ThingIFAPI#getTrigger(String)
- ThingIFAPI#listTrigger(int, String)
- ThingIFAPI#patchCommandTrigger(TriggeredCommandForm, Predicate, TriggerOptions)
- ThingIFAPI#patchServerCodeTrigger(ServerCode, Predicate, TriggerOptions)
- ThingIFAPI#deleteTrigger(String)
- ThingIFAPI#enableTrigger(Boolean)
- ThingIFAPI#listTriggeredServerCodeResults(String, int, String)

### CODE REVIEW

File: thingiftest/src/androidTest/java/com/kii/thingif/largetests/TriggerTest.java [hunk 1]
```diff
@@ -0,0 +1,781 @@
...
+                new AliasAction<>(
+                        ALIAS1,
+                        new AirConditionerActions(true, 25)));
+
+        Condition condition1 = new Condition(new EqualsClauseInTrigger(ALIAS1, "power", true));
+        StatePredicate predicate1 = new StatePredicate(condition1, TriggersWhen.CONDITION_TRUE);
+
+        TriggeredCommandForm form = TriggeredCommandForm.Builder.newBuilder(aliasActions).build();
+
+        Trigger trigger1 = api.postNewTrigger(form, predicate1, null);
```
Comment (shinsuke-nara) [inline thingiftest/src/androidTest/java/com/kii/thingif/largetests/TriggerTest.java]: Using `api.postNewTrigger(form, predicate1)` is better but this is very cosmetic.
You can ignore this pointing out :)

File: thingiftest/src/androidTest/java/com/kii/thingif/largetests/TriggerTest.java [hunk 2]
```diff
@@ -0,0 +1,781 @@
...
+        String thingPassword = "password";
+
+        // on-boarding thing
+        OnboardWithVendorThingIDOptions options =
+                new OnboardWithVendorThingIDOptions.Builder()
+                        .setThingType(DEFAULT_THING_TYPE)
+                        .setFirmwareVersion(DEFAULT_FIRMWARE_VERSION).build();
+        Target target = api.onboardWithVendorThingID(vendorThingID, thingPassword, options);
+        Assert.assertEquals(TypedID.Types.THING, target.getTypedID().getType());
+        Assert.assertNotNull(target.getAccessToken());
```
Comment (shinsuke-nara) [inline thingiftest/src/androidTest/java/com/kii/thingif/largetests/TriggerTest.java]: How about creating `ThingIFAPI` instance and onboarding the instance in `setup()` method?

**Comment by shinsuke-nara** [comment]
@yongpingchen 
`basicTriggerTest` is too large. Could you divide it 3 or 4 test methods?

**Comment by yongpingchen** [comment]
@shinsuke-nara 
> basicTriggerTest is too large. Could you divide it 3 or 4 test methods?
I divide it to 2 test: commandTriggerTest and serverCodeTriggerTest

**Comment by yongpingchen** [comment]
@shinsuke-nara fixed by your comments. Please confirm

**Comment by shinsuke-nara** [comment]
@yongpingchen LGTM!

**Pull request merged**